buildbot/deploy

```diff
@@ -38,13 +39,16 @@ elif [[ "${branch}" =~ ^refs/pull/([0-9]+)/ ]]; then
 
 	# First get the GitHub username of the Pull Request author.
 	pr_number=${BASH_REMATCH[1]}
-	username=`get_pr_username ${pr_number}`
+	pr_data=`get_pr_data ${pr_number}`
+	username=`get_pr_username "${pr_data}"`
+	labels=`get_pr_labels "${pr_data}"`
```

Comment:
I tried to minimize api calls, so first the json is fetched, then it is queried twice.